### PR TITLE
Edit Clean up page in tutorial

### LINKED ISSUE #4671: Reorganise the clean up pages in the tutorial (getting started guide)
## Background

There are a couple of locations where cleanup is mentioned in the getting started tutorial. One at the end of the [Serving section](https://knative.dev/docs/getting-started/first-traffic-split/#clean-up), and the other at the end of the [Eventing section](https://knative.dev/docs/getting-started/clean-up/). 

Users might want to use their cluster to try code samples, but the getting started tutorial tells you to delete your cluster before providing links to what you can do with it next.

## Tasks

- Move all cleanup tasks to one page. The page should include cleaning up Serving resources, cleaning up Eventing resources, and deleting the cluster
- Move the cleanup page after [what's next](https://knative.dev/docs/getting-started/next-steps/) in the site navigation.

### PR DESCRIPTION
**Pull request by snneji**
Fixes #4671

### CODE REVIEW

**Comment by snneji** [comment]
/hold

Waiting for https://github.com/knative/docs/pull/4688 to merge. This will fix the broken link.

File: docs/getting-started/clean-up.md [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
-=== "kn"
-    Delete the Service by running the command:
+We recommend that you delete the cluster used for this tutorial to free up resources
+on your local machine.
 
-    ```bash
-    kn service delete cloudevents-player
-    ```
+If you want to continue experimenting with Knative after deleting the cluster,
+you can reinstall Knative on a new cluster using the [`quickstart` plugin](quickstart-install.md) again.
```
Comment (psschwei) [inline docs/getting-started/clean-up.md:10]: nit: maybe link to the section on running the plugin

```suggestion
you can reinstall Knative on a new cluster using the [`quickstart` plugin](quickstart-install.md/#run-the-knative-quickstart-plugin) again.
```
(I assume that should work for the link, but wasn't able to verify that it actually does)

**Review by psschwei — APPROVED** [review]
/lgtm

**Pull request merged**